crates/xilem_html/src/lib.rs

```diff
@@ -21,13 +22,14 @@ pub use xilem_core::MessageResult;
 pub use app::App;
 pub use class::class;
 pub use context::{ChangeFlags, Cx};
+pub use either::{Either, Either3, Either4, Either5, Either6, Either7, Either8};
```

Comment:
I think a wildcard export is good enough.

Reply:
I strongly dislike wildcard imports / re-exports. This list is easy enough to read as-is

Reply:
Ok, fair point, I have no strong opinion on this (and generally agree with you)